src/main/java/at/ac/tuwien/kr/alpha/solver/TrailAssignment.java

```diff
@@ -198,7 +207,7 @@ private void removeLastDecisionLevel() {
 		int start = trailIndicesOfDecisionLevels.get(getDecisionLevel());
 		ListIterator<Integer> backtrackIterator = trail.listIterator(start);
 		while (backtrackIterator.hasNext()) {
-			Integer backtrackAtom = backtrackIterator.next();
+			int backtrackAtom = atomOf(backtrackIterator.next());
```

Comment:
Please document somewhere in the JavaDocs that the trail is now a trail of literals, not of atoms, but that the interfaces (e.g. `getAssignmentsToProcess`) still handle atoms.

Reply:
Done.